package.json

```diff
@@ -30,19 +33,23 @@
     "jdenticon": "^2.2.0",
     "jitsi-iframe-api": "^1.0.0",
     "lodash-es": "^4.17.15",
+    "prerender-spa-plugin": "^3.4.0",
```

Comment:
As I understand, this package prerenders pages on the build stage. It can't pretender tip page created by the user after publishing the app. We don't need to use this package.

Reply:
I moved it to the devDependencies

Reply:
Removed, due we decide to use `@akryum/vue-cli-plugin-ssr`